ui/pages/institutional/custody/__snapshots__/custody.test.js.snap

```diff
@@ -90,6 +90,7 @@ exports[`CustodyPage renders CustodyPage 2`] = `
         </ul>
       </div>
     </div>
+    
```

Comment:
Nit
```suggestion
```

Reply:
I'll investigate in the future why there's this extra empty line. If I remove it, the snapshot fails. So I'll leave it here. We need to refactor this component and add more tests.